tests/Tus/ServerTest.php

```diff
@@ -1456,6 +1480,18 @@ public function it_returns_100_for_aborted_upload()
             ->with($fileSize)
             ->andThrow(new ConnectionException);
 
+        $this->tusServerMock
+            ->getRequest()
+            ->getRequest()
+            ->headers
+            ->set('Content-Type', 'application/offset+octet-stream');
+
+        $this->tusServerMock
+            ->getRequest()
+            ->getRequest()
+            ->headers
+            ->set('Content-Length', 200);
```

Comment:
instead of 2 `set` statements, we can use single `add` method to set headers at once
```php
$this->tusServerMock
    ->getRequest()
    ->getRequest()
    ->headers
    ->add([
        'Content-Type' => 'application/offset+octet-stream',
        'Content-Length' => 200,
    ]);
```